ui/src/app/base/components/Button/Button.md

```diff
@@ -0,0 +1,73 @@
+## Base
+
+```jsx
+<Button appearance="base">Base</Button>
+<Button appearance="base" disabled={true}>
+  Base disabled
+</Button>
+```
+
+```jsx
+<Button element="a" href="#test" appearance="base">
+  Base link
+</Button>
+<Button element="a" href="#test" appearance="base" disabled={true}>
+  Base link button
+</Button>
+```
+
+## Neutral
+
+```jsx
+<Button appearance="neutral">Neutral button</Button>
+<Button appearance="neutral" disabled={true}>
+  Neutral button disabled
+</Button>
+```
+
+## Positive
+
+```jsx
+<Button appearance="positive">Positive button</Button>
+<Button appearance="positive" disabled={true}>
+  Positive button disabled
+</Button>
+```
+
+## Negative
+
+```jsx
+<Button appearance="negative">Negative button</Button>
+<Button appearance="negative" disabled={true}>
+  Negative button disabled
+</Button>
+```
+
+## Brand
+
+```jsx
+<Button appearance="brand">Brand button</Button>
+<Button appearance="brand" disabled={true}>
+  Brand button disabled
+</Button>
+```
+
+## Inline
+
+```jsx
+<span>Everything you need to get started with Vanilla. </span>
+<Button appearance="neutral" inline={true}>
+  Inline button
+</Button>
+```
+
+## Icon
+
+```jsx
+<Button appearance="p-button" hasIcon={true}>
```

Comment:
Is this correct? Looks like we're passing a vanilla class to this prop?

Reply:
Oops, fixed.